packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -163,24 +171,29 @@ const generateImageAttachmentImgHtml = (message: ChatMessageWithStatus, attachme
     const contentType = extractAttachmentContentTypeFromName(attachment.name);
     const resourceCache = message.resourceCache?.[attachment.previewUrl];
     const src = getResourceSourceUrl(resourceCache);
-
+    // if in error state
+    if (src === undefined) {
+      return `\r\n<p>${getBrokenImageViewNode()}</p>`;
+    }
```

Comment:
For local messages (sent by the user), if images fail to download, and then the user clicked on the "Try sending again" button in the context menu, then should we sending out this broken image node in the content? Or we should remove it from the content before sending.

Reply:
had a discussion with leah offline, the plan is to retain current image tag attributes and when we introduce inline image for "myMessage", it would be converted to image tag upon resending